CHANGELOG.md

```diff
@@ -14,6 +13,7 @@ It is based on Tidepool Blip 1.27.
 - YLP-918 Caregivers bg unit change not taken without reload
 - YLP-919 Yourloops do not encode correctly passwords with special characters
 - YLP-926 Sanitize HTML in message note
+- YLP-931 Leave and delete team when there is patient fail
```

Comment:
a detail, the description is not align with jira:
"Cannot leave and delete a team that still contains a patient"